app/component/itinerary/Navigator.js

```diff
@@ -33,6 +34,19 @@ function Navigator({
 
   return (
     <div>
+      <div className="navigator-top-section">
+        <button
+          type="button"
+          onClick={() => {
+            if (window && window.location) {
+              window.location.reload();
+            }
+          }}
+          className="close-button cursor-pointer"
+        >
+          <Icon img="icon-icon_close" className="medium" />
```

Comment:
Close cross is not black, it probably obeys some theme color (primary_color ?)

Reply:
Clearing style medium off and and new color attribute to navigator-top-section -style.